docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerBuildVariableContributor.java

```diff
@@ -31,9 +31,11 @@ public void buildVariablesFor(AbstractBuild build, Map<String, String> variables
             variables.put("JENKINS_CLOUD_ID", dockerComputer.getCloudId());
             try {
                 //replace http:// and https:// from docker-java to tcp://
-                final String dockerHost = new URIBuilder(dockerComputer.getCloud().serverUrl)
-                        .setScheme("tcp")
-                        .toString();
+                final URIBuilder uriBuilder = new URIBuilder(dockerComputer.getCloud().serverUrl);
+                if (uriBuilder.getScheme() != "unix") {
```

Comment:
I'm new to both Java and Groovy - so tell me, how would it properly look in Java then?